web/modules/custom/portland/modules/portland_openid_connect/portland_openid_connect.module

```diff
@@ -193,44 +97,64 @@ function _get_all_users($access_token)
     'headers' => [
       'Content-Type' => 'application/json',
       'Authorization' => 'Bearer ' . $access_token,
+      'ConsistencyLevel' => 'eventual',
     ],
   ];
 
   try {
     // Run this command to restart the sync process next time
-    // lando drush state:del user_sync.resume_url
-
-    if ($resume_url = $system_state->get('user_sync.resume_url')) {
-      \Drupal::logger('portland OpenID')->notice('Resume retrieving user information ...');
+    // lando drush state:del pgov.user_sync.resume_url.portlandoregon.gov
+    $domain = "portlandoregon.gov";
+    if ($resume_url = $system_state->get("pgov.user_sync.resume_url.$domain")) {
+      \Drupal::logger('portland OpenID')->notice("Resuming user sync for $domain");
     } else {
-      \Drupal::logger('portland OpenID')->notice('Start retrieving user information ...');
+      \Drupal::logger('portland OpenID')->notice("Starting user sync for $domain");
     }
 
+    // Get the user sync batch size. Default is 50 users
+    $batch_size = \Drupal::state()->get('pgov.user_sync.batch_size') ?? "100";
```

Comment:
Is the comment above incorrect? Should it say 100 users?

Reply:
Comment updated